docs/integrations/atlantis-integration.md

```diff
@@ -74,6 +73,40 @@ terrascan server
 
 Once the systems are running, when atlantis is called via pull request, or a comment of `atlantis plan`, terrascan will be called as part of the atlantis plan flow. Scan results will be placed in a comment on the pull request, and if issues are found the test will be marked as failed.
 
-## Custom Atlantis Contaier
-(coming soon...)
+## Custom Atlantis Container
+
+### Usage
+
+To use our container image:
+```
+docker pull accurics/terrascan_atlantis:v0.1.0
```

Comment:
on versioning...

I'd think we version the container image the same as terrascan releases, just to be consistent?

Also here, maybe specify terrascan_atlantis:latest, and refer users to the tags tab of the docker hub page if they want to find specific versions?

Reply:
> I'd think we version the container image the same as terrascan releases, just to be consistent?

then versions of atlantis releases should also be considered?

Reply:
Hmm...I as new versions of helm/terraform/kustomize/etc come out, we usually have to do a new release to support, so probably the same for Atlantis?